common/enclave/cc_data.cpp

```diff
@@ -88,6 +89,17 @@ bool cc_data::build_attested_data(ByteArray& attested_data)
         COND2ERR(!pb_encode_string(&ostream, (const unsigned char*)s.c_str(), s.length()));
     }
 
+    {
+        // NOTE: for the one-chaincode-one-enclave FPC-Lite version, the chaincode encryption key
+        // is serialized directly in the attested data message.
+        // In a multi-enclave scenario, this will be moved to a separate message and flow.
+
```

Comment:
We should explicitly reference that this is a deviation of the specification of FPC lite. 
```suggestion
        // NOTE: for the one-chaincode-one-enclave FPC-Lite version, the chaincode encryption key
        // is serialized directly in the attested data message.
        // This is a (momentary) short-cut over the FPC and FPC Lite specification in `docs/design/fabric-v2+/fpc-registration.puml` and `docs/design/fabric-v2+/fpc-key-dist.puml`

```